cli/cmd/upgrade.go

```diff
@@ -420,6 +420,12 @@ func repairConfigs(configs *pb.All) {
 	if configs.Proxy.DebugImage == nil {
 		configs.Proxy.DebugImage = &pb.Image{}
 	}
+	if configs.GetProxy().GetDebugImage().GetImageName() == "" {
+		configs.Proxy.DebugImage.ImageName = k8s.DebugSidecarImage
+	}
+	if configs.GetProxy().GetDebugImageVersion() == "" {
+		configs.Proxy.DebugImageVersion = version.Version
+	}
```

Comment:
Old versions of control plane did not have the debugImage info in the protobuf, so for example, upgrades from 2.6 -> 2.7 wound up with empty image name and version causing the issue.